src/utils/env_parser.c

```diff
@@ -188,6 +188,63 @@ float parse_env_to_float(const char* env_var)
     return result;
 }
 
+unsigned int parse_env_to_time(const char* env_var) {
```

Comment:
After testing some real cases, I found it problematic to have the time interval's unit to be second. Note that because of the use of `tv_sec`, currently, the minimum interval in sem wait and heartbeat check is 1 second. This is too long for some applications. We need to change that.